independent-publisher-connectors/Givebutter/apiDefinition.swagger.json

```diff
@@ -0,0 +1,5407 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Givebutter",
+    "description": "Givebutter is the world's first completely free end-to-end fundraising platform. Create beautiful giving experiences that feel like magic. Raise more with modern forms, campaigns, and events, and tools. This is fundraising, done right.",

```

Comment:
@troystaylor can you please provide the main purpose and value offered by the connector in the description? More information about requirements and limits for description can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).